src/leap/keymanager/__init__.py

```diff
@@ -339,8 +356,9 @@ def send(pubkey):
                 self._api_uri,
                 self._api_version,
                 self._uid)
-            self._put(uri, data)
+            d = self._put(uri, data)
             emit_async(catalog.KEYMANAGER_DONE_UPLOADING_KEYS, self._address)
```

Comment:
Maybe this should be a callback of _put. I don't know if this event is used anywhere, and if it actually matters, but emits 'done uploading' before know if there was any kind of error uploading them.